src/logger_dispatch_overrides.jl

```diff
@@ -207,3 +207,25 @@ preprocess(name, val::TBVector{T,N}, data) where {T<:Complex,N} =
     push!(data, name*"/re"=>TBVector(real.(content(val))), name*"/im"=>TBVector(imag.(content(val))))
 summary_impl(name, val::TBVector) = histogram_summary(name, collect(0:length(val.data)),
                                                             val.data)
+
+########## Hyperparameters ########################
+
+struct TBHParamsConfig <: WrapperLogType
```

Comment:
What is exactly the use-case of HParamsConfig?

Because I don't think it makes sense to define a wrapper type.
Wrapper types are thought as a way to override the default logging backend for the end-user: for example,  arrays are by default logged as histograms, but you can wrap it in a `TBImage` wrapper and it will be logged as an image.

I would think that `HParamsConfig` is something you can only log to the `hparams` backend so you should just define the `preprocess` and `summary_impl` functions for `HParamsConfig`.